scripts/rollup/bundles.js

```diff
@@ -76,6 +76,44 @@ const bundles = [
     externals: [],
   },
 
+  /******* React Runtime *******/
+  {
+    bundleTypes: [
+      UMD_DEV,
+      UMD_PROD,
+      UMD_PROFILING,
```

Comment:
Did we decide we want to have UMD builds for these? I don't remember.

Reply:
Good point. Just talked to Seb and we should get rid of them because UMD doesn't work with auto import, so there's no point in keeping these